andela/fargo-ah
# #159206049  Implement email verification of newly created users

#### What does this PR do?
- Send verification email to a newly created user or an unverified user
#### Description of Task to be completed?
- write units test for the response on sending, verifying and reverifying a user
- send a verification email to a new user
- verify generated token
- update user status to verified
#### How should this be manually tested?
Have the following endpoints working

 POST /api/users

 GET /api/users/verify

 GET /api/users/recover

 In postman navigate to `http://localhost:3000/api/users`
  
#### Any background context you want to provide?
- Pull in changes and run `npm install`
- Start server with `npm run dev`

#### What are the relevant pivotal tracker stories?
#159206049


[Delivers #159206049]
